classes/Project/Project.mjs

```diff
@@ -0,0 +1,65 @@
+import dbDriver from "../../database/driver.mjs"
+import {validateProjectPayload} from "../../utilities/validatePayload.mjs"
+import {User} from "../User/User.mjs"
+import ProjectFactory from "./ProjectFactory.mjs"
+
+let err_out = Object.assign(new Error(), {
+  status: 500,
+  message: "Unknown Server error"
+})
+
+const database = new dbDriver("mongo")
+
+export default class Project {
+  #creator
+  constructor() {}
+
+  /**
+   * @param {any} userAgent
+   */
+  set creator(userAgent) {
```

Comment:
Moving forward, a `creator` will have two ways of being known: `id` and `agent`. We have cleverly aligned them such that `https://store.rerum.io/v1/id/hexadecimal` has the `_id` "hexadecimal".

So what should we be storing here? What would a developer want in the Project being used for services? Ultimately, the `creator` attached to the saved annotations will be the full agent, but that will be automatically added by the API and never rely on the contributor to add.

Reply:
I believe the creator on the project is better off being an `agent`. Especially since the `id` can be extracted from the same if needed. Maybe we shouldn't be setting creator for this. we have access to the user's token so we can add the agent from it to the project payload when a project is being created.

Reply:
Hash, not URI. At least 2:1. Sorry Victor

Reply:
deprecate user.agent to user._id

Reply:
Think about 

https://store.rerum.io/v1/id/62572ba71d974d1311abd673

vs

http://localhost:3009/user/62572ba71d974d1311abd673

We don't want the user-creator to be https://store.rerum.io/v1/id/62572ba71d974d1311abd673 because that is not a representation of the TPEN user.